firefox-ios/Shared/Strings.swift

```diff
@@ -1577,7 +1577,7 @@ extension String {
                     value: "Bottom",
                     comment: "On the toolbar customization onboarding card, the string used to describe the option to set the toolbar at the bottom of the screen.")
                 public static let ContinueAction = MZLocalizedString(
-                    key: "Onboarding.Modern.Customization.Theme.Continue.Action.v123",
+                    key: "Onboarding.Modern.Customization.Theme.Continue.Action.v140",
```

Comment:
Hey! Just stumbled on this. This string was already exported and only changing the key will effectively remove all translations for that string. Why are we changing the key here?